pint_server/models.py

```diff
@@ -81,24 +82,26 @@ class ProviderServerBase(PintBase):
 
 class AmazonImagesModel(Base, ProviderImageBase):
     __tablename__ = 'amazonimages'
+    #__table_args__ = (UniqueConstraint('id', name='amazonimages_uniq_id'),)
```

Comment:
Should this commented out line be removed? Or does it serve some "documentation" purpose? If the latter then perhaps include a preamble comment with relevant details.

Reply:
I can remove that code. As it turned out the `id` field in `amazonimages` table is already unique. I commented it out during testing but forgot to remove it.

Reply:
Done